test/EFCore.PG.FunctionalTests/Query/SpatialQueryNpgsqlGeographyTest.cs

```diff
@@ -207,6 +210,12 @@ public override async Task ToText(bool isAsync)
         public override Task StartPoint(bool isAsync)                => Task.CompletedTask;
         public override Task SymmetricDifference(bool isAsync)       => Task.CompletedTask;
         public override Task Touches(bool isAsync)                   => Task.CompletedTask;
+
+        [ConditionalTheory]
+        [MemberData(nameof(IsAsyncData))]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "xUnit1026:Theory methods should use all of their parameters")]
```

Comment:
No real need for this, although if you really want to we can suppress for the entire file.